RecoPixelVertexing/PixelTriplets/plugins/CAHitNtupletGeneratorKernels.cu

```diff
@@ -270,11 +270,13 @@ void CAHitNtupletGeneratorKernelsGPU::classifyTuples(HitsOnCPU const &hh, TkSoA
     kernel_fillHitInTracks<<<numberOfBlocks, blockSize, 0, cudaStream>>>(tuples_d, quality_d, device_hitToTuple_.get());
     cudaCheck(cudaGetLastError());
   }
-  if (m_params.minHitsPerNtuplet_ < 4) {
```

Comment:
need to fix line 262 as well
https://github.com/cms-patatrack/cmssw/blob/cfa4d27436eba2728b415b36d98b7372a2de9c91/RecoPixelVertexing/PixelTriplets/plugins/CAHitNtupletGeneratorKernels.cu#L262

Reply:
eventually check all comparisons with m_params.minHitsPerNtuplet_

Reply:
this is Major

Reply:
Checked around and added it also in the .cc for `kernel_countHitInTracks` and `kernel_fillHitInTracks`